OneCommunityGlobal/HGNRest
# Diego putting back unit and integration tests

# Description
Brings back unit and integration test files

## Related PRS (if any):
N/A

## Main changes explained:
N/A

## How to test:
1. check into current branch
2. do `npm install` and `...` to run this PR locally
3. run the following command to run unit and integration tests: `npx jest --passWithNoTests --silent --noStackTrace --runInBand --forceExit`
4. 
## Screenshots or videos of changes:

## Note:
